sklearn/mixture/dpgmm.py

```diff
@@ -501,10 +501,10 @@ def fit(self, X):
             List of n_features-dimensional data points.  Each row
             corresponds to a single data point.
         """
-        self.random_state = check_random_state(self.random_state)
+        self.random_state_ = check_random_state(self.random_state)
```

Comment:
Why? How else should we keep sampling? If we do ``check_random_state`` before each sampling, it will yield the same result on each call, right?